xbmc/profiles/ProfileManager.cpp

```diff
@@ -127,12 +127,6 @@ void CProfileManager::OnSettingsLoaded()
   CDirectory::Create(URIUtils::AddFileToFolder(strDir,"mixed"));
 }
 
-void CProfileManager::OnSettingsSaved() const
-{
-  // save mastercode
-  Save();
-}
```

Comment:
This change (thats in leia branch already) has reportedly broken any profile settings being saved

https://github.com/xbmc/xbmc/issues/17412

follow up PR to revert the leia change by user who submitted the issue

https://github.com/xbmc/xbmc/pull/17413

Any insight into why that may be fritsch? Is this really meant to be removed, or an oversight?